### PR TITLE
Update development section

### PR DESCRIPTION
**Pull request by AlicjaSuska**
Updating the 'development' section with clearer instructions and updated file path

readme-file-path

### CODE REVIEW

**Comment by felixfbecker** [comment]
Maybe highlight the `chrome` folder in the last section of the screenshot too?

**Comment by AlicjaSuska** [comment]
Updated screenshot:

Screenshot 2020-10-22 at 12 23 32

**Comment by AlicjaSuska** [comment]
@felixfbecker could you please review? Thank you!

File: client/browser/README.md [hunk 1]
```diff
@@ -80,12 +80,12 @@ Now, follow the steps below for the browser you intend to work with.
 ### Chrome
 
 - Browse to [chrome://extensions](chrome://extensions).
-- If you already have the Sourcegraph extension installed, disable it by unchecking the "Enabled" box.
-- Click on [Load unpacked extensions](https://developer.chrome.com/extensions/getstarted#unpacked), and select the `build/chrome` folder.
+- If you already have the Sourcegraph extension installed, disable it using the toggle.
+- Enable 'developer mode', click on [Load unpacked extensions](https://developer.chrome.com/extensions/getstarted#unpacked), save it in the `sourcegraph/client/browser/build/chrome` folder.
 - Browse to any public repository on GitHub to confirm it is working.
-- After making changes it is necessary to refresh the extension. This is done by going to [chrome://extensions](chrome://extensions) and clicking "Reload".
+- After making changes it is necessary to refresh the extension. This is done by going to [chrome://extensions](chrome://extensions) and clicking the "Reload" icon.
```
Comment (felixfbecker) [inline client/browser/README.md:86]:
```suggestion
- After making changes it is sometimes necessary to refresh the extension. This is done by going to [chrome://extensions](chrome://extensions) and clicking the "Reload" icon.
```

**Review by felixfbecker — COMMENTED** [review]
Thanks for improving this!

**Pull request merged**